src/scss/components/search/_search.scss

```diff
@@ -0,0 +1,30 @@
+@charset "UTF-8";
+
+@include grid-media-min('md') {
+  .su-site-search {
+    display: none;
+  }
+
+  .su-multi-menu {
+    display: flex;
+  
+    .menu-search-button {
+      position: unset;
+      margin-left: 3.6rem;
+      align-self: center;
+      width: 2.6rem;
```

Comment:
The search icon appears a bit bigger than the neighboring text

![image](https://github.com/SU-SWS/ppo_irsr_subtheme/assets/34019925/22387815-15c1-442f-a8d4-e447d326cc47)

Would you be able to update the sizing to match the figma mock? 😃 

![image](https://github.com/SU-SWS/ppo_irsr_subtheme/assets/34019925/999e6731-c927-430a-a482-9b9b6a5cc00c)